src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/SupportApiServiceImpl.scala

```diff
@@ -0,0 +1,196 @@
+import it.pagopa.interop.backendforfrontend.service.TenantProcessService
+import it.pagopa.interop.backendforfrontend.model._
+import it.pagopa.interop.tenantprocess.client.{model => TenantProcess}
+import it.pagopa.interop.backendforfrontend.error.BFFErrors._
+import it.pagopa.interop.backendforfrontend.error.Handlers.handleError
+import it.pagopa.interop.commons.utils.service.OffsetDateTimeSupplier
+import it.pagopa.interop.commons.utils.TypeConversions._
+import it.pagopa.interop.commons.utils._
+import javax.xml.parsers.DocumentBuilderFactory
+import org.w3c.dom
+import org.opensaml.saml2.core.Response
+import org.opensaml.security.SAMLSignatureProfileValidator
+import org.opensaml.xml.validation.ValidationException
+import org.opensaml.xml.{XMLObject, Configuration}
+import org.opensaml.DefaultBootstrap
+import org.opensaml.xml.io.{Unmarshaller, UnmarshallerFactory}
+
+import cats.syntax.all._
+import java.time.{OffsetDateTime, Instant, ZoneOffset}
+import java.io.ByteArrayInputStream
+import scala.jdk.CollectionConverters._
+import scala.concurrent.{Future, ExecutionContext}
+import scala.util.Success
+
+final case class SupportApiServiceImpl(
+  sessionTokenGenerator: SessionTokenGenerator,
+  tenantProcessService: TenantProcessService,
+  offsetDateTimeSupplier: OffsetDateTimeSupplier
+)(implicit ec: ExecutionContext)
+    extends SupportApiService {
+
+  private implicit val logger: LoggerTakingImplicit[ContextFieldsToLog] =
+    Logger.takingImplicit[ContextFieldsToLog](this.getClass)
+
+  val SUPPORT_LEVELS: Seq[String] = Seq("L2", "L3")
+  val SUPPORT_LEVEL_NAME: String  = "supportLevel"
+  val OPERATOR: String            = "OPERATOR"
+
+  override def redirectToSupportPage(
+    sAMLResponse: SAMLResponse
+  )(implicit contexts: Seq[(String, String)], toEntityMarshallerProblem: ToEntityMarshaller[Problem]): Route = {
+
+    logger.info(s"Calling Support SAML")
+
+    val result: Future[(String, String)] = for {
+      _            <- validate(parseResponse(sAMLResponse.response))
+      tenant       <- ApplicationConfiguration.pagoPaTenantId.toFutureUUID >>= tenantProcessService.getTenant
+      selfcareId   <- tenant.selfcareId.toFuture(MissingSelfcareId(tenant.id))
+      sessionToken <- sessionTokenGenerator.generate(
+        signatureAlgorithm = SignatureAlgorithm.RSAPkcs1Sha256,
+        claimsSet = buildClaims(selfcareId, tenant),
+        audience = ApplicationConfiguration.generatedJwtAudience,
+        tokenIssuer = ApplicationConfiguration.generatedJwtIssuer,
+        validityDurationInSeconds = 300
+      )
+      base64       <- sAMLResponse.response.encodeBase64.toFuture
+    } yield (base64, sessionToken)
+
+    onComplete(result) {
+      handleError(s"Error calling support SAML") orElse { case Success((base64, sessionToken)) =>
+        val redirectUrl =
+          s"https://selfcare.dev.interop.pagopa.it/ui/it/assistenza/scelta-ente#saml2=$base64&jwt=$sessionToken"
```

Comment:
you need to turn this url into a configuration.
This value is valid only for dev